src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -5,16 +5,21 @@
  */
 public class DoctorConstants {
     public static final String AADHAAR_ALREADY_PRESENT = "Aadhaar already present in database";
-    public static final String AADHAAR_NUMBER_PATTERN_MESSAGE = "Aadhaar number must be only 12 digits!";
     public static final String AADHAAR_REGEX_PATTERN = "^[2-9]{1}[0-9]{11}$";
     public static final String AADHAAR_SHOULD_BE_MANDATORY = "Aadhaar should be mandatory!";
+    public static final String AADHAAR_SHOULD_NOT_EMPTY = "Aadhaar should not empty!";
```

Comment:
AADHAAR_SHOULD_NOT_EMPTY = "Aadhaar should not empty!"
change it to AADHAAR_SHOULD_NOT_BE_EMPTY = "Aadhaar should not be empty!"

Reply:
Changed to AADHAAR_SHOULD_NOT_BE_EMPTY